WcaOnRails/lib/database_dumper.rb

```diff
@@ -0,0 +1,447 @@
+        "guests" => :keep,
+        "id" => :keep,
+        "ip" => "''",
+        "updated_at" => :keep,
+        "user_id" => "''",
+      }.freeze,
+    }.freeze,
+    "schema_migrations" => {
+      public_condition: "",
+      column_sanitizers: {
+        "version" => :keep,
+      }.freeze,
+    }.freeze,
+    "team_members" => {
+      public_condition: "",
+      column_sanitizers: {
+        "created_at" => :keep,
+        "end_date" => :keep,
+        "id" => :keep,
+        "start_date" => :keep,
+        "team_id" => :keep,
+        "team_leader" => :keep,
+        "updated_at" => :keep,
+        "user_id" => :keep,
+      }.freeze,
+    }.freeze,
+    "teams" => {
+      public_condition: "",
+      column_sanitizers: {
+        "created_at" => :keep,
+        "description" => :keep,
+        "friendly_id" => :keep,
+        "id" => :keep,
+        "name" => :keep,
+        "updated_at" => :keep,
+      }.freeze,
+    }.freeze,
+    "user_preferred_events" => {
+      public_condition: "",
+      column_sanitizers: {
+        "event_id" => :keep,
+        "id" => :keep,
+        "user_id" => :keep,
+      }.freeze,
+    }.freeze,
+    "users" => {
+      public_condition: "",
+      column_sanitizers: {
+        "avatar" => :keep,
+        "confirmation_sent_at" => "NULL",
+        "confirmation_token" => "NULL",
+        "confirmed_at" => :keep,
+        "country_iso2" => :keep,
+        "created_at" => :keep,
+        "current_sign_in_at" => :keep,
+        "current_sign_in_ip" => "''",
+        "delegate_id_to_handle_wca_id_claim" => :keep,
+        "delegate_status" => :keep,
+        "dob" => "'1954-12-04'",
+        "email" => "CONCAT(id, '@worldcubeassociation.org')",
+        "encrypted_password" => "''",
+        "gender" => :keep,
+        "id" => :keep,
+        "last_sign_in_at" => :keep,
+        "last_sign_in_ip" => "''",
+        "location_description" => :keep,
+        "name" => :keep,
+        "notes" => "''",
+        "pending_avatar" => "NULL",
+        "phone_number" => "NULL",
+        "preferred_locale" => "NULL",
+        "region" => :keep,
+        "remember_created_at" => "NULL",
+        "reset_password_sent_at" => "NULL",
+        "reset_password_token" => "NULL",
+
+        "results_notifications_enabled" => :keep,
+        "saved_avatar_crop_h" => :keep,
+        "saved_avatar_crop_w" => :keep,
+        "saved_avatar_crop_x" => :keep,
+        "saved_avatar_crop_y" => :keep,
+        "saved_pending_avatar_crop_h" => :keep,
+        "saved_pending_avatar_crop_w" => :keep,
+        "saved_pending_avatar_crop_x" => :keep,
+        "saved_pending_avatar_crop_y" => :keep,
+        "senior_delegate_id" => :keep,
+        "sign_in_count" => "1",
+        "unconfirmed_email" => "NULL",
+        "unconfirmed_wca_id" => :keep,
+        "updated_at" => :keep,
+        "wca_id" => :keep,
+      }.freeze,
+    }.freeze,
+    "vote_options" => :discard,
+    "votes" => :discard,
+  }.freeze
```

Comment:
This could be a lot more compact and readable. Maybe something like this:

    some_table: {
      keep: %w(avatar confirmed_at country_iso2 created_at current_sign_in_at etc),
      NULL: %w(confirmation_sent_at confirmation_token),
      discard: %w(completed_jobs delayed_jobs delegate_reports),
      db_default: %w(notes current_sign_in_ip),
      fake_values: {
        dob: "'1954-12-04'",
        email: "CONCAT(id, '@worldcubeassociation.org')",
      }
    }

...or maybe this is fine and I'm overdesigning.